src/ray/object_manager/pull_manager.cc

```diff
@@ -179,6 +186,11 @@ void PullManager::DeactivateUntilWithinQuota(
   }
 }
 
+// TODO(ekl) this doesn't take into account bytes used by already-local objects. This
```

Comment:
```suggestion
// TODO(ekl) this doesn't take into account bytes used by pinned, already-local objects. This
```

Reply:
Could they be non-pinned?

Reply:
I don't think so, the object store reports availability according to objects in use.